client/jetpack-cloud/sections/partner-portal/lib/get-product-title.ts

```diff
@@ -5,8 +5,8 @@
  * @returns Product title
  */
 export default function getProductTitle( product: string, removeVariant: boolean = false ): string {
-	if ( 'Jetpack AI' === product ) {
-		return 'AI Assistant';
+	if ( 'Jetpack AI' === product || 'Jetpack AI Assistant' === product ) {
+		return 'AI';
 	}
```

Comment:
I'm okay with committing these lines now, but let's make a note to make this more robust. Any change to the name of the product will make the comparison useless.